regression/verilog/casts/static_cast1.desc

```diff
@@ -0,0 +1,8 @@
+KNOWNBUG
```

Comment:
Is this still correct? If so, could you please add a comment at the end

Reply:
Ah, no, actually works, will fix